yarn-project/noir-libs/value-note/src/utils.nr

```diff
@@ -80,4 +181,75 @@ fn send_note(
         encryption_pub_key,
         note.serialise(),
     );
+}
+
+/*
+  Sends three amounts to three recipients.
+  Why three? Because one private call currently allows `MAX_NEW_COMMITMENTS_PER_CALL = 4` output commitments.
+  So we split the output notes as: 3 to recipients + 1 to the owner (the change note).
+*/
+fn send_notes(
```

Comment:
I personally feel this shouldn't be in `noir-lib` but in `multi_transfer` because no one else would need it but also IDM either way